### ISSUE TITLE
Remote branch 3.2.14 not found in upstream origin

### ISSUE THREAD

**Issue by vbmota** [PopcornTimeTV/PopcornTimeTV#375]
after pod install:
[!] Error installing PopcornKit
[!] /usr/bin/git clone https://github.com/PopcornTimeTV/PopcornKit.git /var/folders/dd/ht212v_135qgvgnl9nh699680000gn/T/d20170301-966-1a45cr1 --template= --single-branch --depth 1 --branch 3.2.14

Cloning into '/var/folders/dd/ht212v_135qgvgnl9nh699680000gn/T/d20170301-966-1a45cr1'...
warning: Could not find remote branch 3.2.14 to clone.
fatal: **Remote branch 3.2.14 not found in upstream origin**

LATEST version of master!

**Comment by FlaviaBondJamesBondsHusband** [comment]
#374

**Comment by stepo11** [comment]
I am also getting this error. I have downloaded latest master and still getting it.

**Comment by stepo11** [comment]
Is there a command in terminal to reset or clean the podfile?

**Comment by FlaviaBondJamesBondsHusband** [comment]
`git reset --hard; git pull; pod repo update; pod install`

Please use the install script if you have other issues.

**Comment by stepo11** [comment]
Still getting same issue...

`Stephens-MBP:PopcornTimeTV stephen$ git pull
Already up-to-date.
Stephens-MBP:PopcornTimeTV stephen$ git reset --hard
HEAD is now at e6e5991 Add issuetemplate.md
Stephens-MBP:PopcornTimeTV stephen$ pod update
Update all pods
Updating local specs repositories
Analyzing dependencies
Pre-downloading: `Reachability` from `https://github.com/tonymillion/Reachability`
Downloading dependencies
Installing 1PasswordExtension (1.8.4)
Installing AEXML (4.0.2)
Installing Alamofire (4.2.0)
Installing AlamofireImage (3.2.0)
Installing AlamofireNetworkActivityIndicator (2.1.0)
Installing AlamofireXMLRPC (2.1.0)
Installing CSStickyHeaderFlowLayout (0.2.11-1)
Installing FloatRatingView (2.0.1)
Installing GCDWebServer (3.3.3)
Installing Locksmith (3.0.0)
Installing MBCircularProgressBar (0.3.5)
Installing MarqueeLabel (3.0.1)
Installing MobileVLCKit-unstable (3.0.0a24)
Installing OBSlider (1.1.1)
Installing ObjectMapper (2.2.5)
Installing PopcornKit (3.2.14)

[!] Error installing PopcornKit
[!] /usr/bin/git clone https://github.com/PopcornTimeTV/PopcornKit.git /var/folders/r8/8pbjl9cx3bg_74wfy6_s1flc0000gn/T/d20170301-8230-tppbfh --template= --single-branch --depth 1 --branch 3.2.14

Cloning into '/var/folders/r8/8pbjl9cx3bg_74wfy6_s1flc0000gn/T/d20170301-8230-tppbfh'...
warning: Could not find remote branch 3.2.14 to clone.
fatal: Remote branch 3.2.14 not found in upstream origin

Stephens-MBP:PopcornTimeTV stephen$ 
`

**Comment by FlaviaBondJamesBondsHusband** [comment]
@stepo11 No idea why that's happening as you can clearly see that the master Podfile points to 3.2.13 not 3.2.14 (https://github.com/PopcornTimeTV/PopcornTimeTV/blob/master/Podfile#L9). Can you delete the directory and re-download master?

**Comment by gustavolessadublin** [comment]
Just tested. Same issue, same error message.
I've been trying for a couple of hours to make subtitles appear again using the latest master (freshly downloaded) and a backup I made at the moment when everything was working.
Both are giving me the same error (when using pod repo update; pod install).

**Comment by Dewinster** [comment]
Hi,

I had the same problem aswell and I think it's because the 3.2.14 version is still defined in the Specs. (Not sure if this could is the real problem.) https://github.com/PopcornTimeTV/Specs/tree/master/PopcornKit

A temporarily solution could be to change following line (they occur twice; once in "def pods" and once in "target TopShelf") in the **Podfile**:
`pod 'PopcornKit', '~> 3.2.13'`
to 
`pod 'PopcornKit', '3.2.13'`

This will force it to download the 3.2.13 version and skip all higher versions.
After that the project build perfectly with me, but now I am stuk with a runtime error. :)

**Comment by FlaviaBondJamesBondsHusband** [comment]
@Dewinster Huh, yeah I thought I had it deleted. Guess not. Can you try again with the original spec?

**Comment by Dewinster** [comment]
@FlaviaBondJamesBondsHusband Yeah it builds perfectly, but there are some more issues going on with the project. But I do not have much time to have a closer look into it.
